internal/tag.go

```diff
@@ -15,10 +15,32 @@ const tagToken = "{{tag}}"
 
 var spaces = regexp.MustCompile(`\s+`)
 
+func (c *Converter) tagList(note *enex.Note, md *markdown.Note) []byte {
```

Comment:
Actually, it can be dropped completely. I try to keep the signature of every conversion step consistent, and `addFrontMatter` method achieves the goal. `tagList` looks like a helper that extracts tags from the note into a slice of bytes, so it has no reason to access the markdown note.
```suggestion
func (c *Converter) tagList(note *enex.Note) []byte {
```